### PR TITLE
#161676743 User Logout Feature

### PR DESCRIPTION
**Pull request by obulaworld**
#### What does this PR do?
 - Implemented the logout feature

#### Description of Task to be completed?
- a user should be able to log out of the application

#### How should this be manually tested?
- start the app by running ```yarn start:dev```
the app should open on port ```8080``` on your browser
navigate to the login page, login as a user then use the logout 
link to log out of the app

#### What are the relevant pivotal tracker stories?
[#161676743](https://www.pivotaltracker.com/story/show/161676743, "Users should be able to Logout of Authors Haven")

### CODE REVIEW

File: src/components/Logout/Logout.jsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+/**
+ * @desc renders home page
+ * @param {props} props
+ * @return {object} redirect
+ */
+const Auth = (props) => {
+  localStorage.removeItem('authorsHavenAuthToken');
+  localStorage.removeItem('user');
+  props.logout();
+  console.log(props);
```
Comment (Mrshinezee) [inline src/components/Logout/Logout.jsx]: Please, remove the console.log

File: src/components/Logout/__test__/logout.test.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+// React Libraries
+import React from 'react';
+import { Provider } from 'react-redux';
+
+// third-party libraries
+import { shallow, configure } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+
+// modules importation
+import Logout from '../Logout';
```
Comment (Mrshinezee) [inline src/components/Logout/__test__/logout.test.js:10]: comment with
//components

//state

File: src/components/Logout/__test__/logout.test.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+import configureStore from '../../../store/configureStore';
+
+configure({ adapter: new Adapter() });
+
+describe('Login Component', () => {
+  test('renders the Logout Component', () => {
+    const store = configureStore();
+    const wrapper = shallow(<Provider store={store}><Logout /></Provider>);
```
Comment (Mrshinezee) [inline src/components/Logout/__test__/logout.test.js]: You can break it down to
```
<Provider store={store}>
<Logout />
</Provider>
```

**Review by andela-phemmy — APPROVED** [review]
Looking Good To Me. Nice Work.

**Review by Mrshinezee — CHANGES_REQUESTED** [review]

File: src/components/Logout/Logout.jsx [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+  return (
+    <Redirect to='/' />
+  );
+};
+
+Auth.propTypes = {
+  logout: PropTypes.func.isRequired
+};
+const mapDispatchToProps = dispatch => ({
+  logout: () => {
```
Comment (Mrshinezee) [inline src/components/Logout/Logout.jsx]: You can make a one line
```logout: () => dispatch(logOutUser())```

**Review by Mrshinezee — CHANGES_REQUESTED** [review]

**Review by KaiserPhemi — APPROVED** [review]

**Pull request merged**